### PR TITLE
Feature/new click banner

### PR DESCRIPTION
**Pull request by punalsc**
Hi @gtvj ,

Could you have a look at this PR.

Thank you.

### CODE REVIEW

**Comment by gtvj** [comment]
Hi @punalsc - I don't think `one.js`, `two.js` and `logger.js` are needed in this repo. I'll remove them.

**Comment by gtvj** [comment]
Hi @punalsc - are the development instructions in `README.md` out of date now? It looks like Grunt has been removed from the project.

File: src/bannerClick.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import {promoObjFunc} from './modules/promoObj';
+
+let banner = document.getElementById('imageviewerOverlay');
+if (banner) {
+    banner.addEventListener('click', promoObjFunc('promotionClick', 'bts', 'Image viewer banner', 'Banner', 'On watermarked image'), true);
```
Comment (gtvj) [inline src/bannerClick.js]: Hi @punalsc - Is this right? Here you are passing a _call_ to `promoObjFunc()` as the second argument to `addEventListener`. I'm used to the second argument being a function reference. I think that passing it as a call will result in the function being called once (when the JavaScript first loads) and the returned object being provided as the handler to be invoked when the user clicks on the element. See here: 
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#The_event_listener_callback

Comment (punalsc) [reply src/bannerClick.js]: Good spotting. This is now fixed.

**Comment by punalsc** [comment]
Hi @gtvj , The `one.js`, `two.js` and `logger.js` were left in there for reference, and was going to clear it out in another feature branch along with an updated `Readme`.

**Pull request merged**